src/format/stream/stream.rs

```diff
@@ -27,6 +27,12 @@ impl<'a> Stream<'a> {
 		}
 	}
 
+	pub fn codec_parameters(&self) -> codec::Parameters {
```

Comment:
I just wanted to avoid `parameters` being understood as stream parameters.
But now I feel it won't be a problem because it's return type is `codec::Parameters`.
I will change it.